<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>1362</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>398</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="focus_wheel_horizontalLayout">
     <item>
      <widget class="QCheckBox" name="focus_wheel_connected_checkBox">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>focus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="focus_wheel_position_doubleSpinBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="focus_wheel_backward_pushButton">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="focus_wheel_jog_doubleSpinBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="focus_wheel_foreward_pushButton">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="focus_wheel_target_position_doubleSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="additional_widgets"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="toupcam_connected_checkBox">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>toupcam</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="toupcam_live_checkBox">
       <property name="text">
        <string>live</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toupcam_show_ui_pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>show ui</string>
       </property>
       <property name="styleSheet">
        <string notr="true">color:blue</string>
       </property>
       <property name="text">
        <string>ui</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gridGroupBox">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Beam Path</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="shutter_connected_checkBox">
        <property name="text">
         <string>shutter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="flip_mirror_connected_checkBox">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>flip mirror</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="flip_mirror_position_checkBox">
        <property name="text">
         <string>Spec / APD</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="shutter_open_comboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="focus_wheel_connected_checkBox_2">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>focus</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="power_wheel_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Power Wheel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <item>
       <widget class="QCheckBox" name="power_wheel_connected_checkBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="power_wheel_0_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="power_wheel_90_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>90</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="power_wheel_180_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>180</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="power_wheel_270_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>max</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="power_wheel_jog_forward_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="power_wheel_jog_doubleSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="power_wheel_jog_backward_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="power_wheel_position_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>22</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Position</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="power_wheel_target_position_doubleSpinBox">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="power_wheel_zero_encoder_pushButton">
          <property name="text">
           <string>zero</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="power_meter_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Power Meter</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="power_meter_connected_checkBox">
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="power_meter_activation_pushButton">
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="power_meter_wavelength_doubleSpinBox">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="power_meter_show_ui_pushButton">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>show ui</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color:blue</string>
          </property>
          <property name="text">
           <string>ui</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="power_meter_power_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Power </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gridGroupBox">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Spectrum</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="Spectrometer">
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <item row="0" column="5">
       <widget class="QPushButton" name="picam_show_ui_pushButton">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:blue</string>
        </property>
        <property name="text">
         <string>ui</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="picam_connected_checkBox">
        <property name="text">
         <string>picam</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QDoubleSpinBox" name="picam_exposuretime_doubleSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="spectrometer_connected_checkBox">
        <property name="text">
         <string>spectrometer</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QDoubleSpinBox" name="picam_count_rate_doubleSpinBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="picam_continuous_checkBox">
        <property name="maximumSize">
         <size>
          <width>13</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>c</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QDoubleSpinBox" name="spectrometer_center_wavelength_doubleSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gridGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>APD</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>25</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="apd_connected_checkBox">
        <property name="maximumSize">
         <size>
          <width>15</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>APD</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="apd_count_rate_doubleSpinBox">
        <property name="font">
         <font>
          <family>Gadugi</family>
          <pointsize>16</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="apd_show_ui_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>show ui</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color:blue</string>
        </property>
        <property name="text">
         <string>ui</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
